--- origsrc/edb-1.0.5.042/configure.in	2008-01-24 20:56:46.000000000 -0600
+++ src/edb-1.0.5.042/configure.in	2008-06-11 23:01:15.125000000 -0500
@@ -233,6 +233,7 @@
 dnl Some architectures require DB to be compiled with special flags and/or
 dnl libraries for threaded applications
 case "$host_os" in
+cygwin*)  CPPFLAGS="-D_REENTRANT $CPPFLAGS";;
 freebsd*) CPPFLAGS="-D_THREAD_SAFE -pthread $CPPFLAGS";;
 irix*)	  CPPFLAGS="-D_SGI_MP_SOURCE $CPPFLAGS";;
 linux*)	  CPPFLAGS="-D_REENTRANT $CPPFLAGS";;
--- origsrc/edb-1.0.5.042/src/Makefile.am	2006-09-04 19:36:44.000000000 -0500
+++ src/edb-1.0.5.042/src/Makefile.am	2008-06-11 23:01:15.171875000 -0500
@@ -54,4 +54,4 @@
 uts4_cc.s
 
 libedb_la_DEPENDENCIES = $(top_builddir)/config.h
-libedb_la_LDFLAGS      = -version-info 1:5:0
+libedb_la_LDFLAGS      = -version-info 1:5:0 -no-undefined
